classes/security/authorization/PublicationWritePolicy.inc.php

```diff
@@ -32,6 +32,10 @@ function __construct($request, &$args, $roleAssignments) {
 		// Is the user assigned to this submission in one of these roles, and does this role
 		// have access to the _current_ stage of the submission?
 		$this->addPolicy(new StageRolePolicy([ROLE_ID_MANAGER, ROLE_ID_SUB_EDITOR, ROLE_ID_ASSISTANT, ROLE_ID_AUTHOR]));
+
+		// Is publication editable by user
+		$currentUser = $request->getUser();
+		$this->addPolicy(new PublicationCanBeEditedPolicy($currentUser, 'api.submissions.403.userCantEdit'));
```

Comment:
Can you pass in the `$request` instead of the `$currentUser` just to keep this policy consistent with others?

Reply:
Fixed